src/App.tsx

```diff
@@ -1,14 +1,59 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+enum FilterType {
+  Active = 'active',
+  Completed = 'completed',
+  All = 'all',
+}
+
+const filterTodos = (todos: Todo[], filterType: string, filterText: string) => {
+  let filteredTodos: Todo[] = [...todos];
+
+  if (filterType === FilterType.Active) {
+    filteredTodos = filteredTodos.filter(todo => {
+      return !todo.completed;
+    });
+  }
+
+  if (filterType === FilterType.Completed) {
```

Comment:
- `filter` doesn't mutate any array so you don't need to create a copy
- Use switch case instead